react/src/components/summoner/MatchFilterForm.tsx

```diff
@@ -32,9 +32,15 @@ export function MatchFilterForm({onUpdate}: {onUpdate: (data?: MatchFilterFormTy
   }, [])
 
   return (
+
     <>
       <form>
-        <div className="input-field dark">
+        <div 
+          style={{
+            width:600,
+          }} 
+          className="input-field dark"
```

Comment:
In my editor, I'm seeing some trailing whitespace on 39 and 41, can you remove those?